std_msgs/Header header

string object_id
string roi_type
string roi_subtype

geometry_msgs/Twist motion
BoundingBoxMsg bounding_box

string[] sensor_ids